Scripts/RemediationScripts/Remediate-SecureFTPDeploymentForAppServices.ps1

```diff
@@ -495,7 +522,7 @@ function Enable-SecureFTPDeploymentForAppServices
                 return
               }
             }
-
+        }
         Write-Host "[Step 4 of 4] Enable Secure FTP Deployments for App Services"
         Write-Host $([Constants]::SingleDashLine)
         # To hold results from the remediation.
```

Comment:
line 541: Is that else block necessary, aren't the users are restricted to choose either FTPSOnly or Disabled?

Reply:
Closed as per discussion in the call.